src/components/pets/Favorites.tsx

```diff
@@ -70,34 +72,100 @@ export default function Favorites() {
     dispatch(removeFavoritePet(removedPet.id));
   };
 
+  function filterList(petType: string) {
+    let filtered;
+    if (petType === "All-favorites") {
+      filtered = petList;
+    } else {
+      filtered = petList?.filter((pet) => pet?.type === petType);
+    }
+    return filtered;
+  }
+
+  const handleClick = (e: any) => {
```

Comment:
Can u fix the lint issue

Reply:
Both on neovim and Vscode there is no listing issue showing.

Reply:
solved the lint issue